src/identity/components/GlobalHeader/GlobalHeaderDropdown/GlobalHeaderDropdown.scss

```diff
@@ -70,3 +70,19 @@
   margin: 6px 0;
   background-color: rgba(241, 241, 243, 0.1);
 }
+
+.cf-dropdown-item {
```

Comment:
I don't think you need to include the `.cf-dropdown-item` selector as long as the upgrade-payg-add-org--buton is unique.